iwf/workflow_state_options.py

```diff
@@ -87,14 +114,25 @@ def _to_idl_state_options(
         res.execute_api_retry_policy = options.execute_api_retry_policy
     if options.execute_api_timeout_seconds is not None:
         res.execute_api_timeout_seconds = options.execute_api_timeout_seconds
-    if options.execute_failure_handling_state is not None:
+    if options.execute_failure_handling_state_when_retry_exhausted is not None:
```

Comment:
Should we make the naming more consistent like in the JavaSDK?
```
wait_until_api_failure_policy_when_retry_exhausted
execute_api_failure_policy_when_retry_exhausted
```
?

Reply:
Technically it should be `retries` exhausted

Reply:
yeah I am thinking on that too. We can't do exactly because python sdk is using a differetn model. 

Maybe:
* proceed_to_state_after_execute_retry_exhausted
* proceed_to_execute_after_wait_until_retry_exhausted

Our java SDK is :
* setProceedWhenExecuteRetryExhausted
* setProceedWhenWaitUntilRetryExhausted

WDYT?

Reply:
Makes sense to me 👍

Reply:
Done